abdulfatai360/banka
# #165049076 Set Up Linting Library and Style Guide

#### What does this PR do?
Set up linting framework and enforces code style guide

#### Description of Task to be completed?
- Have ESLint installed as a project dependency
- Set Airbnb as the choice style guide

#### How should this be manually tested?
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo by running `cd banka/` on terminal
- Ensure your working directory is clean by running  `git status`  command on terminal
- Get a copy of the PR by typing  `git fetch origin pull/16/head:ch-setup-linting-165049076`
- Now that you have a copy of the branch, switch to it using  `git checkout ch-setup-linting-165049076`
- Run `npm install` in terminal to install project's dependencies
- Your directory will now be an exact copy of the PR.

#### What are the relevant pivotal tracker stories?
#165049076
